scripts/generateTsConfigsInApi.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
-const { getPackages, getPackage } = require("./utils/getPackages");
-const { relative } = require("path");
+const { getPackages, getPackage, PROJECT_ROOT } = require("./utils/getPackages");
```

Comment:
What is this file? Why do we need this? Was this a temporary script or it's something we might need in the future? 🤔

Reply:
@Pavel910 told me we have `tsconfigs` generated and I found, that we do it in this file. It adds typescript project references and `compilerOptions.paths` basing on `package.json` references.

Reply:
As I mentioned before, I had to fix few things here, because extended config paths were wrong sometimes.

Reply:
@doitadrian it's similar to `generageTsConfigsInPackages` :) I was sick of `api` folder pointing to declaration files when I Cmd+Click the import path.